src/pages/tickets/org/orgHeader/index.tsx

```diff
@@ -413,39 +414,45 @@ const StatusContainer = styled.div<styledProps>`
 
 const Status = ['Open', 'Assigned', 'Completed', 'Paid'];
 export const OrgHeader = ({
+  bypassAsyncCheck = false,
```

Comment:
is this value only used for testing purposes? I think it would make more sense to mock the response we need to show it in the test than leaking testing implementation details into the actual codebase

Reply:
I understand your concern about keeping the production code clean. The `bypassAsyncCheck`was indeed for testing. I'll remove it from the production code and use mocking in our tests instead. Thanks for pointing this out!